packages/protvista-structure/src/protvista-structure.js

```diff
@@ -103,58 +112,87 @@ class ProtvistaStructure extends HTMLElement {
     return this.setAttribute("accession", accession);
   }
 
-  get isResidueOnlyHighlight() {
-    return this.hasAttribute("highlightresidues");
+  get hideTable() {
+    return this.getAttribute("hide-table");
+  }
+
+  get molecule() {
+    return this.getAttribute("molecule");
+  }
+
+  get height() {
+    return this.getAttribute("height");
+  }
+
+  get hideViewportControls() {
```

Comment:
I don't think this is used?

Reply:
No, it is used in line 16:
    this._height = this.height || "480px";

`this.height` is the getter where line 124 is the body of the function.